extension-manifest-v2/src/classes/Common.js

```diff
@@ -84,7 +85,7 @@ common.load = async () => {
 
 	setPref('modules.anolysis.enabled', false);
 
-	if (!IS_ANDROID) {
+	if (!IS_ANDROID && HAS_WASM) {
```

Comment:
Maybe
```
if (IS_ANDROID) {
		await setModuleState('hpn-lite', enabled);
		await setModuleState('human-web-lite', enabled);
} else if (HAS_WASM) {
		await setModuleState('hpnv2', enabled);
		await setModuleState('human-web', enabled);
}
```

Rationale: only hpnv2 depends on wasm. And common/human-web-lite de-facto is a pure Android implementation.

So: if Android -> human-web-lite
otherwise, use human-web+hpnv2 unless the lack of wasm introduce a performance bug in hpnv2